src/earn/saga.ts

```diff
@@ -312,22 +314,22 @@ export function* withdrawSubmitSaga(action: PayloadAction<WithdrawInfo>) {
         networkId,
         type: TokenTransactionTypeV2.EarnWithdraw,
         inAmount: {
-          value: amount,
+          value: pool.balance,
           tokenId,
         },
         outAmount: {
-          value: amount,
-          tokenId: networkConfig.aaveArbUsdcTokenId,
+          value: pool.balance,
+          tokenId: pool.dataProps.withdrawTokenId,
```

Comment:
nit: unpack data props above when getting `depositTokenId`. Could also just unpack the pool object into all the necessary parts so don't need `pool.` for each of them.